types/firestore.d.ts

```diff
@@ -613,12 +616,28 @@ declare namespace FirebaseFirestore {
   }
 
   /**
-   * An options object that can be used to disable request throttling in
-   * BulkWriter.
+   * An options object to configure throttling on BulkWriter.
    */
   export interface BulkWriterOptions {
-    /** Whether to disable throttling. */
+    /**
+     * Whether to disable throttling. If set, initialOpsPerSecond and
+     * maxOpsPerSecond cannot be set.
+     */
     readonly disableThrottling?: boolean;
+
```

Comment:
A lot of the APIs I have been dealing with lately do the following:

```
export interface BulkWriterOptions {
readonly throttling?: boolean : {initialOpsPerSecond?: number; maxOpsPerSecond?: number; };
}
```

This would match terser and eslint, but we might want to gather more feedback (maybe from @feiyang1).

Reply:
attempted